src/omv/modules/py_sensor.c

```diff
@@ -656,6 +692,24 @@ static mp_obj_t py_sensor_ioctl(uint n_args, const mp_obj_t *args) {
         }
         #endif
 
+        #if (OMV_ENABLE_OV5640 == 1 || OMV_ENABLE_OV7725 == 1)
```

Comment:
```suggestion
```
Those should not be gated by sensor type.

Reply:
Done